frontend/resources/include/GUIRenderRequest.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * GUIRenderRequest.h
+ *
+ * Copyright (C) 2020 by VISUS (Universitaet Stuttgart).
+ * Alle Rechte vorbehalten.
+ */
+
+#pragma once
+
+#include "WindowCollection.h"
+typedef megamol::gui::WindowCollection::WindowConfiguration WinConfig_t;
```

Comment:
`WindowCollection.h` includes `GUIUtils.h` which includes 

```
#include "imgui.h"
#include "imgui_impl_opengl3.h"
#include "imgui_internal.h"
#include "imgui_stdlib.h"

#include "glm/glm.hpp"
#include "glm/gtc/matrix_transform.hpp"
#include "glm/gtc/type_ptr.hpp"
```

Is there a way to keep this resource minimal regarding the dependencies? If the Header files of Services and other things get blown up by such includes our compilation times may suffer. Is there a way to wrap `WinConfig_t` functionality locally and hide such header includes in a .cpp implementation file?

Reply:
see commit b3fe3c4